tests/factory_tests.py

```diff
@@ -73,6 +73,16 @@ def test_one_arg_timestamp(self):
         with self.assertRaises((OverflowError, ValueError)):
             self.factory.get(timestamp)
 
+    def test_one_arg_timestamp_with_tzinfo(self):
+
+        int_timestamp = int(time.time())
```

Comment:
May be better to keep this as a float timestamp: `timestamp = time.time()`. Makes sure to test all aspects of the timestamp.

Reply:
All done!